eve/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker: &master-worker
+      type: docker
+      path: eve/workers/master
+      volumes:
+        - '/home/eve/workspace'
+    steps:
+      - Git:
+          name: fetch source
+          repourl: '%(prop:git_reference)s'
+          shallow: 'True'
```

Comment:
You can remove the quote if you want to.

Reply:
The linter doesn't like it :p